packages/babel-preset-env/test/fixtures/corejs3/exclude-regenerator/output.mjs

```diff
@@ -33,7 +33,6 @@ import "core-js/modules/es.array.map.js";
 import "core-js/modules/es.array.of.js";
 import "core-js/modules/es.array.reduce.js";
 import "core-js/modules/es.array.reduce-right.js";
-import "core-js/modules/es.array.reverse.js";
```

Comment:
Interesting, after specifying `ie 6` as targets, `es.array.reverse` is removed!

Reply:
I think it was included due to Chrome